BLE_SecurityAndPrivacy/source/main.cpp

```diff
@@ -513,33 +497,73 @@ class PrivacyCentral : public PrivacyDevice {
                 _is_connecting = true;
 
                 return;
-
             }
         }
     }
 
 private:
-    bool _is_connecting;
+    bool _is_connecting = false;
 };
 
-/* only seed the random number generation once per application run */
-bool PrivacyDevice::_seeded = false;
+#if MBED_CONF_APP_FILESYSTEM_SUPPORT
```

Comment:
I wonder if we shouldn't have help code for setting up file system.

Reply:
I don't think this comes under connectivity, maybe we can ask core